app/src/main/res/values-fr/strings.xml

```diff
@@ -108,6 +108,8 @@
     <string name="erase_options">Tout effacer</string>
     <string name="ride">Tour</string>
     <string name="recent_activities">Activités récentes</string>
+    <string name="share_title">Lasta : Partage d\'activité</string>
+    <string name="share_body">Hello ! Regardes cette activité sur Lasta :</string>
```

Comment:
The verb "regarder" to the impératif tense is "regarde" without an s at the end :)

Reply:
Also corrected, thank you